data/src/emulator.js

```diff
@@ -263,6 +263,8 @@ class EmulatorJS {
         })();
         this.canvas = this.createElement('canvas');
         this.canvas.classList.add('ejs_canvas');
+        this.videoRotation = (typeof this.config.videoRotation === "number") ? this.config.videoRotation : 0;
```

Comment:
Should we also assert this is one of 0, 90, 180, 270, or is there a use case to set the game at 45? Just wondering what you think - not necessarily requesting changes here

Reply:
I thought to add just 0, 90, 180, 270
But having the option to set any degree could be useful.
So it could be -45 or -130 if there ever was a need.
`videoRotation` is set as the default option in the menu